js-envs/src/main/scala/org/scalajs/jsenv/rhino/ScalaJSCoreLib.scala

```diff
@@ -163,8 +160,8 @@ class ScalaJSCoreLib private[rhino] (semantics: Semantics,
     val linkedClass = providers.getOrElse(encodedName,
         throw new ClassNotFoundException(encodedName))
 
-    val desugared = new ScalaJSClassEmitter(semantics, ECMAScript51Global,
-        linkingUnit).genClassDef(linkedClass)
+    val desugared = new ScalaJSClassEmitter(ECMAScript51Global, linkingUnit)
+      .genClassDef(linkedClass)
```

Comment:
You use twice the same style, here and at line 108-109. I suggest the one from 108-109.